src/Faker/Provider/fa_IR/PhoneNumber.php

```diff
@@ -36,8 +45,9 @@ class PhoneNumber extends \Faker\Provider\PhoneNumber
         '0939#######',
         '0920#######',
         '0921#######',
-        '0937#######',
-        '0937#######',
+        '0922#######',
```

Comment:
please order alphabetically, and provide a format specification in the comments.